google/cloud/spanner/integration_tests/backup_integration_test.cc

```diff
@@ -130,6 +137,11 @@ TEST_F(BackupTest, BackupTest) {
       << "Database " << db.database_id()
       << " not found in the backup operation list.";
 
+  // TODO(#4306) - CreateBackup() times out (rarely) log for troubleshoot
+  if (backup_future.wait_for(std::chrono::minutes(5)) != fs::ready) {
+    GCP_LOG(WARNING) << "Potential timeout in CreateBackup()" << db;
```

Comment:
Nit: Could probably do with a space between `CreateBackup()` and the database name (et seq).

Reply:
Fixed.